packages/accounts-password/password_server.js

```diff
@@ -193,8 +193,7 @@ Accounts.registerLoginHandler("password", options => {
   // First the login is validated, then the code situation is checked
   if (
     !result.error &&
-    Accounts._is2faEnabledForUser &&
-    Accounts._is2faEnabledForUser(options.user)
+    Accounts._check2faEnabled(user)
```

Comment:
You need to check if the function `Accounts._check2faEnabled` exists here, the same way we're doing with the `Accounts._is2faEnabledForUser`.

 If you don't have the package `accounts-2fa` on your project, the function `Accounts._check2faEnabled` will be undefined.

Reply:
True. Another way would be to check if the package is installed, but this way it could also be turned into a feature by allowing third party packages to override this function if they follow the same structure. :thinking:

Reply:
Added a conditional here: `Accounts._check2faEnabled?.(user)` one less line, but if you think that added a line with expressly checking the line is better, I'll change to that.